setup.cfg

```diff
@@ -47,14 +49,14 @@ install_requires =
     future>=0.17.1
     tensorboard>=2.2.0
     PyYAML>=5.1
-    pytorch_lightning>=1.7.6
+    pytorch_lightning>=1.7.6,<=1.9.5
```

Comment:
`pytorch-lightning==2.0.0` breaks some things in `cultionet`

Reply:
I saw pytorch-lightning==2.0.0 in early March 👍